scripts/wrapLatestCommitMsg.fsx

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System.IO
+open System
+
+#load "../src/FileConventions/Library.fs"
+
+#r "nuget: Fsdk, Version=0.6.0--date20230214-0422.git-1ea6f62"
+
+open Fsdk
+open Fsdk.Process
+
+let commitMsg =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "log -1 --format=%B"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+        .Trim()
+
+let header, mayBeBody =
+    let newLineIndex = commitMsg.IndexOf Environment.NewLine
+
+    if newLineIndex > 0 then
+        commitMsg.Substring(0, newLineIndex).Trim(),
+        Some(commitMsg.Substring(newLineIndex).Trim())
+    else
+        commitMsg, None
+
+let maxCharsPerLine = 64
+
+let wrappedBody =
+    match mayBeBody with
+    | Some body -> FileConventions.WrapText body maxCharsPerLine
+    | _ -> ""
+
+let newCommitMsg =
+    header
+    + Environment.NewLine
+    + Environment.NewLine
+    + wrappedBody
+    + Environment.NewLine
+
+File.WriteAllText("wlog.txt", newCommitMsg)
```

Comment:
@realmarv this is wrong, because the previous message we gave to the user involved dealing with this wlog.txt, but then now you tell the user to call this script and you don't tell him what to do with this file

Reply:
I added the git amend command. BTW I'll squash when we're done with this script

Reply:
What about not creating a file at all?

Reply:
that would be even better indeed; if you find a way to do that, great

Reply:
git amend has a -m option for defining the commit message directly

Reply:
Great let's use it

Reply:
Done

Reply:
tested?